subprojects/libft/src/ft_list_unlink.c

```diff
@@ -1,23 +1,28 @@
 /* ************************************************************************** */
 /*                                                                            */
 /*                                                        ::::::::            */
-/*   ft_lstadd.c                                        :+:    :+:            */
+/*   ft_list_unlink.c                                   :+:    :+:            */
 /*                                                     +:+                    */
-/*   By: nloomans <nloomans@student.codam.nl>         +#+                     */
+/*   By: nloomans <nloomans@student.codam.n>          +#+                     */
 /*                                                   +#+                      */
-/*   Created: 2019/03/29 13:44:04 by nloomans       #+#    #+#                */
-/*   Updated: 2019/03/29 13:49:37 by nloomans      ########   odam.nl         */
+/*   Created: 2020/02/21 19:51:24 by nloomans      #+#    #+#                 */
+/*   Updated: 2020/02/21 19:51:25 by nloomans      ########   odam.nl         */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "libft.h"
 
-/*
-** TODO: Remove this function. It simply exists because it's required to get
-** the bonus points.
```

Comment:
Finally accomplished your 2019 Todo's i see

Reply:
🎉 

I didn't solve it the way I thought I would when I wrote the `TODO` but it's still a victory in my book.